conans/test/unittests/client/build/cmake_test.py

```diff
@@ -1348,6 +1348,34 @@ def test_vcvars_applied(self, generator, compiler, version):
             cmake.build()
             self.assertTrue(vcvars_mock.called, "vcvars weren't called")
 
+    @parameterized.expand([('Ninja',),
+                           ('NMake Makefiles',),
+                           ('NMake Makefiles JOM',),
+                           ('Unix Makefiles',),
+                           ])
+    def test_compilervars_applied(self, generator):
```

Comment:
This test seems a bit shallow for the amount of code. It would be better to check at least the commands generated by the ``compilerargs_command`` too.

Reply:
added more tests for command line